pandas/core/frame.py

```diff
@@ -10485,7 +10485,7 @@ def quantile(
         self,
         q=0.5,
         axis: Axis = 0,
-        numeric_only: bool = True,
+        numeric_only: bool = False,
```

Comment:
If we want to change this, you have to add a FutureWarning that the default will change in a future version

Reply:
Thanks for the feedback, I will work on this!